ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1971,6 +1971,12 @@ router_multicast:
     routing: < true | false >
     multipath: < none | deterministic | "deterministic color" | "deterministic router-id" >
     software_forwarding: < kernel | sfe >
+    rpf:
+      routes:
+        - subnet: < IPv4 subnet / netmask >
```

Comment:
Sorry again to ask to change the model. This word subnet doesn't sound better because we used the keyword destinations.

So do you think the keyword ```source``` sound better here?

Reply:
`source` or maybe `source_prefix` matches the EOS CLI:
```
avd-veos-leaf1(config-s-b-router-multicast-ipv4)#rpf route ?
  A.B.C.D    Source prefix
  A.B.C.D/E  Source address with prefix
```

Reply:
done thanks!